etc/a.gwf

```diff
@@ -43,10 +43,15 @@ highlight_color=#2f6400
 
 # Welcome logo. The designated file should be in bases/src/mybase/images
 # Called with <img src="%prefix;m=IM;v=welcome_logo"
-#   	             style="welcome_logo_style">
+#                  style="welcome_logo_style">
 #welcome_logo=
 #welcome_logo_style=
 
+# Aletrnate text for the main Notes button on the Welcome page
+# adding </div><div class="d-flex justify-content-center mt-2"> at the end
+# will isolate this button above the others
```

Comment:
What is it?

This is very template-specific and has nothing to do here.

Reply:
Yes, to be removed